src/core/client/admin/helpers/getQueueConnection.ts

```diff
@@ -0,0 +1,23 @@
+import { ConnectionHandler, RecordSourceSelectorProxy } from "relay-runtime";
+
+type Queue = "reported" | "pending" | "unmoderated" | "rejected";
+
+export default function getQueueConnection(
+  queue: Queue,
+  store: RecordSourceSelectorProxy
+) {
+  const root = store.getRoot();
+  if (queue === "rejected") {
+    return ConnectionHandler.getConnection(root, "RejectedQueue_comments", {
+      filter: { status: "REJECTED" },
```

Comment:
Was it possible here to use the typescript enum instead of a string constant here?

Reply:
Not yet, we'll get there :-)